src/components/Login/Login.tsx

```
@@ -4,15 +4,15 @@ import { InputWindow } from '../InputSection/InputWindow';
 import { signinUser } from '@/api/userAuth';
 import { useRouter } from 'next/navigation';
-import { HideToggle } from '../Toggle/HideToggle';
-import { useStore } from '@/store/useAuthStore';
+import { useAuthStore } from '@/store/useAuthStore';
 import { getUserData } from '@/api/getUserData';
+import { HideToggleButton } from '../InputSection/HideToggleButton';
 
 const Login = () => {
 	const router = useRouter();
 	const [referrer, setReferrer] = useState<string | null>(null); // referrer 상태 추가
 	const { isLoggedIn, token, userId, setIsLoggedIn, setToken, setUserId } =
```

ESLint

[error] 'isLoggedIn' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)